### ISSUE TITLE
Fleet - add caBundle field to add a private CA certificate to a GitRepo

### ISSUE THREAD

**Issue by izaac** [rancher/dashboard#2083]
fleet:v0.3.2-rc5 adds support to handle git repositories using custom certificates using the `caBundle`.

We should add this new field to the gitRepo UI

https://github.com/rancher/fleet/issues/150
https://github.com/rancher/fleet/issues/182
http://fleet.rancher.io/gitrepo-add/

**Comment by StrongMonkey** [comment]
Also the insecureSkipTLS flag.

**Comment by StrongMonkey** [comment]
caBundle needs to be base64 encoded.

**Comment by izaac** [comment]
Rancher version `v2.5-head (12/22/2020)` _bd660e1_

I tested this using a private Gitlab instance with self signed certificates. I added the CA with the option `Specify additional certificates to be accepted` 

The certificate is automatically base64 encoded and used properly.
I also used the Accept any certificate (insecure) option. Which also worked with the private Gitlab.

The default is require a valid certificate always like the case of Github repos pulled using https.
